app/models/feedback.py

```diff
@@ -5,15 +5,20 @@ class Feedback(db.Model):
     """Feedback model class"""
     __tablename__ = 'feedback'
     id = db.Column(db.Integer, primary_key=True)
-    rating = db.Column(db.String, nullable=False)
+    rating = db.Column(db.Float, nullable=False)
     comment = db.Column(db.String, nullable=True)
     user_id = db.Column(db.Integer,
                         db.ForeignKey('users.id', ondelete='CASCADE'))
     event_id = db.Column(db.Integer,
                          db.ForeignKey('events.id', ondelete='CASCADE'))
 
     def __init__(self, rating=None, comment=None, event_id=None, user_id=None):
-        self.rating = rating
+        rating = float(rating)
+        if not 0 < rating < 5:
+            raise ValueError("Rating should be in range [0, 5].")
+        else:
+            self.rating = round(rating*2, 0) / 2  # Rounds to nearest 0.5
```

Comment:
This is still manual checking rather than marshmallow

Reply:
@schedutron You need to add validations on feedback schema as well, which will allow throwing proper response based on these limits.

Reply:
@poush Do you intend the code to be added here: `open-event-server/app/api/schema/feedbacks.py`?

Reply:
Yes